<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1">


<!--
Font-awesome icons ie github or twitter
-->
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/brands.css" integrity="sha384-n9+6/aSqa9lBidZMRCQHTHKJscPq6NW4pCQBiMmHdUCvPN8ZOg2zJJTkC7WIezWv" crossorigin="anonymous">

<!--
Google fonts api stuff
-->



<title>Combining polygenic risk scores with gene-based burden scores</title>





<style>
@page {
size: 45in 38in;
margin: 0;
padding: 0;
}
body {
margin: 0;
font-size: 45px;
width: 45in;
height: 38in;
padding: 0;
text-align: justify;
font-family: Palatino;
}
.poster_wrap {
width: 45in;
height: 38in;
padding: 0cm;
}
.title_container {
width: 45in;
height: calc(38in * 0.15);
overflow: hidden;
background-color: #0b4545;
border: 0 solid #0b4545;
}
.logo_left {
float: left;
width: 10%;
height: 100%;
background-color: #0b4545;
display: flex;
align-items: center;
justify-content: center;
}
.logo_right {
float: right;
width: 10%;
height: 100%;
background-color: #0b4545;
display: flex;
align-items: center;
justify-content: center;
}
.poster_title {
text-align: center;
position: relative;
float: left;
width: 80%;
height: 100%;
color: #FFFFFF;
top: 50%;
transform: translateY(-50%);
-webkit-transform: translateY(-50%);
}
#title {
font-family: Palatino;
}
/* unvisited link */
a:link {
color: #cc0000;
}
.mybreak {
  break-before: column;
}
/* visited link */
a:visited {
color: #cc0000;
}

/* mouse over link */
a:hover {
color: #cc0000;
}

/* selected link */
a:active {
color: #cc0000;
}
.poster_body {
-webkit-column-count: 3; /* Chrome, Safari, Opera */
-moz-column-count: 3; /* Firefox */
column-count: 3;
-webkit-column-fill: auto;
-moz-column-fill: auto;
column-fill: auto;
-webkit-column-rule-width: 1mm;
-moz-column-rule-width: 1mm;
column-rule-width: 1mm;
-webkit-column-rule-style: dashed;
-moz-column-rule-style: dashed;
column-rule-style: dashed;
-webkit-column-rule-color: #0b4545;
-moz-column-rule-color: #0b4545;
column-rule-color: #0b4545;
column-gap: 1em;
padding-left: 0.5em;
padding-right: 0.5em;
height: 100%;
color: #000000
background-color: #ffffff;
}
.poster_title h1 {
font-size: 75pt;
margin: 0;
border: 0;
font-weight: normal;
}
.poster_body_wrap{
width: calc(45in + 0 + 0);
height: calc(38in * 0.83);
padding-top: calc(38in * 0.005);
padding-bottom: calc(38in * 0.01);
background-color: #ffffff;
}
.poster_title h3 {
color: #ffffff;
font-size: 50pt;
margin: 0;
border: 0;
font-weight: normal;
}
.poster_title h3 > sup {
  font-size: 35pt;
  margin-left: 0.02em;
}
.poster_title h5 {
color: #FFFFFF;
font-size: 35pt;
margin: 0;
border: 0;
font-weight: normal;
}
img {
margin-top: 2cm;
margin-bottom: 0;
}
.section {
  padding: 0.2em;
}
.poster_body h1 {
text-align: center;
color: #FFFFFF;
font-size: 65pt;
border: 2mm solid #0b4545;
background-color: #0b4545;
border-radius: 4mm 0mm;
margin-top: 2mm;
margin-bottom: 2mm;
font-weight: normal;
}
.poster_body h2 {
color: #000000;
font-size: 40pt;
padding-left: 4mm;
font-weight: normal;
}
.span {
width: 200%;
}
/* center align leaflet map,
from https://stackoverflow.com/questions/52112119/center-leaflet-in-a-rmarkdown-document */
.html-widget {
margin: auto;
position: sticky;
margin-top: 2cm;
margin-bottom: 2cm;
}
.leaflet.html-widget.html-widget-static-bound.leaflet-container.leaflet-touch.leaflet-fade-anim.leaflet-grab.leaflet-touch-drag.leaflet-touch-zoom {
position: sticky;
width: 100%;
}
pre.sourceCode.r {
background-color: #dddddd40;
border-radius: 4mm;
padding: 4mm;
width: 75%;
/* align-items: center; */
margin: auto;
padding-left: 2cm;
}
code.sourceCode.r{
background-color: transparent;
font-size: 20pt;
border-radius: 2mm;
}
.caption {
font-size: 25pt;
}
.table caption {
font-size: 25pt;
padding-bottom: 3mm;

}
code {
font-size: 1em;
font-family: monospace;
background-color: #00808024;
color: #0b4545;
padding: 1.2mm;
border-radius: 2mm;
}
.poster_title code {
font-size: 1em;
}
table {
font-size: 40px;
margin: auto;
border-top: 3px solid #666;
border-bottom: 3px solid #666;
}
table thead th {
border-bottom: 3px solid #ddd;
}
td {
padding: 8px;
}
th {
padding: 15px;
}
caption {
margin-bottom: 10px;
}
.poster_body p {
margin-right: 4mm;
margin-left: 4mm;
margin-top: 6mm;
margin-bottom: 10mm;
}
.poster_body ol {
margin-right: 4mm;
margin-left: 4mm;
}
#ul {
margin-right: 4mm;
margin-left: 4mm;
}
.references p {
font-size: 20pt;
}
.orcid img {
  width: 1em;
}
</style>
</head>
<body>


<div class="poster_wrap">
<div class="title_container">
<!-- Left Logo  -->
<div class="logo_left">
<img src= 'Figures/unilogo.png' style="width: 80%">
</div>
<!-- Poster Title -->
<div class= "poster_title">
<h1 id="title">Combining polygenic risk scores with gene-based burden scores</h1>
<h3 id="author">E. Hassanin<sup>1,2</sup>, R. Aldisi<sup>1</sup>, P. May<sup>2,3</sup>, P. Krawitz<sup>1</sup>, DR Bobbili<sup>2</sup>, C. Maj<sup>1</sup></h3><br>
<h5 id="affiliation"><sup>1</sup> Institute of Genomic Statistics and Bioinformatics, University of Bonn, Bonn, Germany<br> <sup>2</sup> MeGeno S.A, Esch-sur-Alzette, Luxembourg<br> <sup>3</sup> Bioinformatics Core, Luxembourg Centre for Systems Biomedicine (LCSB), Belvaux, Luxembourg</h5>
</div>
<!-- Right Logo  -->
<div class="logo_right">
<img src=Figures/megeno.png style="width: 80%">
</div>
</div>

<div class='poster_body_wrap'>
<div class='poster_body'>
<div id="introduction" class="section level1">
<h1>Introduction</h1>
<ol style="list-style-type: decimal">
<li><p>In complex traits ”common diseases”, how a large number of genes associated variants contributes to influence development of the disease over a life course?.</p></li>
<li><p>Polygenic risk score (PRS) uses weighted sum of trait-associated alleles (<strong>common variant</strong>) to calculate an overall risk score of getting a particular common disease.</p></li>
<li><p>Gene-level scoring is based on the combined effects of multiple variants weighted according to functional annotations and variant frequency and deleteriousness<span class="citation"><sup>1</sup></span> (<strong>rare variants</strong>).</p></li>
</ol>
<div id="objectives" class="section level2">
<h2>Objectives</h2>
<p>In this study, we explores whether the integration of PRS and Gene-level scoring can improve risk prediction or no.</p>
</div>
</div>
<div id="methods" class="section level1">
<h1>Methods</h1>
<div id="data-source" class="section level2">
<h2><strong>Data Source</strong></h2>
<p>This study was performed using genotypic and phenotypic data from the Ukbiobank (UKB).</p>
</div>
<div id="study-participants" class="section level2">
<h2><strong>Study participants</strong></h2>
<p>We studied three common diseases: Breast Cancer, CAD, and type 2 diabetes. Diagnosis of the three disease was based self-reported, ICD-9, ICD-10, and OPCS-4 codes.<span class="citation"><sup>2</sup></span></p>
</div>
<div id="statistical-model" class="section level2">
<h2><strong>Statistical model</strong></h2>
<p>we developed a combined model to integrated PRS, and gene-based score.</p>
<p><img src="Figures/prs-genepy.png" width="75%" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="results" class="section level1">
<h1>Results</h1>
<p>
 
</p>
<p>For each disease, we calculated PRS based on previously reported GWAS, and gene-based scores based on trait-associated genes.</p>
<p>
 
</p>
<table>
<caption><span id="tab:table1">Table 1: </span>PRS derivation and gene-based scores for three common, complex diseases.</caption>
<thead>
<tr class="header">
<th align="left">Disease</th>
<th align="center">Discovery GWAS (n)</th>
<th align="center">Discovery genes (n)</th>
<th align="center">Prevalence in training (cases/controls)</th>
<th align="center">Prevalence in testing (cases/controls)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Breast Cancer</td>
<td align="center">228,951</td>
<td align="center">920</td>
<td align="center">779/16,751</td>
<td align="center">259/5,583</td>
</tr>
<tr class="even">
<td align="left">CAD</td>
<td align="center">184,305</td>
<td align="center">74</td>
<td align="center">861/31,537</td>
<td align="center">315/10,483</td>
</tr>
<tr class="odd">
<td align="left">Type 2 diabetes</td>
<td align="center">159,208</td>
<td align="center">53</td>
<td align="center">717/31,681</td>
<td align="center">238/10,560</td>
</tr>
</tbody>
</table>
<p>
 
</p>
<p>Here, we analyzed genotyping and exome data from UK Biobank to perform a combined evaluation of polygenic risk based on common variants and the burden of a rare deleterious mutation in the coding region for three diseases. The derived genetic risk models were assessed on the basis of their ability to classify individuals diagnosed with breast cancer, CAD, and diabetes in the UKB testing dataset.</p>
<p>For breast cancer, the derived genetic risk models from both components reached an AUC of 0.738, whereas from only a polygenic risk score was 0.687, and from only an exome score was 0.728. (Figure <a href="#fig:figure2">1</a>)</p>
<div class="figure"><span id="fig:figure2"></span>
<img src="Figures/bc.png" alt="Breast cancer models. On left hand side, AUC of PRS, gene-based score (gps), and combined model was determined using a logistic regression model adjusted for age, sex, and the first four principal components of ancestry. On right hand side, AUC of PRS, gene-based score (gps) adjusted only for the first four principal components of ancestry. " width="50%" height="100%" /><img src="Figures/bc2.png" alt="Breast cancer models. On left hand side, AUC of PRS, gene-based score (gps), and combined model was determined using a logistic regression model adjusted for age, sex, and the first four principal components of ancestry. On right hand side, AUC of PRS, gene-based score (gps) adjusted only for the first four principal components of ancestry. " width="50%" height="100%" />
<p class="caption">
Figure 1: Breast cancer models. On left hand side, AUC of PRS, gene-based score (gps), and combined model was determined using a logistic regression model adjusted for age, sex, and the first four principal components of ancestry. On right hand side, AUC of PRS, gene-based score (gps) adjusted only for the first four principal components of ancestry.
</p>
</div>
<p>For coronary artery disease, models reached AUC of 0.818, 0.806, and 0.790, while for type 2 diabetes AUC reached 0.750, 0.711, and 0.692 for the combined model, only polygenic risk score, and only exome score, respectively. That suggests that both components associate in an independent way to the phenotypes.(Figure <a href="#fig:figure3">2</a>, <a href="#fig:figure4">3</a>)</p>
<div class="figure"><span id="fig:figure3"></span>
<img src="Figures/cad.png" alt="Coronary artery disease (CAD) models" width="50%" height="100%" /><img src="Figures/cad2.png" alt="Coronary artery disease (CAD) models" width="50%" height="100%" />
<p class="caption">
Figure 2: Coronary artery disease (CAD) models
</p>
</div>
<div class="figure"><span id="fig:figure4"></span>
<img src="Figures/diabetes.png" alt=" Type 2 diabetes models" width="50%" /><img src="Figures/diabetes2.png" alt=" Type 2 diabetes models" width="50%" />
<p class="caption">
Figure 3:  Type 2 diabetes models
</p>
</div>
</div>
<div id="conclusion" class="section level1">
<h1>Conclusion</h1>
<p>1.The genetic contributions to human traits can be associated with both common variant and rare variants.</p>
<ol start="2" style="list-style-type: decimal">
<li>A better evaluation of the genetic susceptibility can be obtained with a combined evaluation of PRS and gene-based burden scores.</li>
</ol>
</div>
<div id="next-steps" class="section level1">
<h1>Next Steps</h1>
<p>This study was performed on ~50,000 samples that have both genotyping and exome sequence data in the UKB. Next step is to reproduce the same work on recently releases 200,000 exome-sequence sample in the UKB.</p>
</div>
<div id="find-out-more" class="section level1">
<h1>Find out more</h1>
<ul>
<li><a href="https://github.com/EmadHassanin/combining_prd_gps" class="uri">https://github.com/EmadHassanin/combining_prd_gps</a></li>
<li><a href="https://www.igsb.uni-bonn.de/en/team-1">Institute of Genomic Statistics and Bioinformatics, University</a>.</li>
<li><a href="https://www.megeno.com/">MeGeno S.A, Company</a>.</li>
</ul>
</div>
<div id="references" class="section level1 unnumbered">
<h1>References</h1>
<div id="refs" class="references">
<div id="ref-Ashton2018">
<p>1. Ashton, J., Pengelly, R., Beattie, R. &amp; MacArthur, B. GenePy – a score for estimating gene pathogenicity in individuals using next-generation sequencing data. <em>bioRxiv</em> 336701 (2018). doi:<a href="https://doi.org/10.1101/336701">10.1101/336701</a></p>
</div>
<div id="ref-khera2019">
<p>2. V.Khera, M.Chaffin, G. Aragam, E. Haas, C.Roselli, S. Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations Amit. <em>Physiology &amp; behavior</em> <strong>176,</strong> 139–148 (2019).</p>
</div>
</div>
</div>
</div>
</div>

</div>



</body>
</html>
